generator/generator_tests/generator_tests.pro

```diff
@@ -30,9 +30,12 @@ SOURCES += \
     osm_id_test.cpp \
     osm_o5m_source_test.cpp \
     osm_type_test.cpp \
+    restriction_collector_test.cpp \
+    restriction_test.cpp \
     source_data.cpp \
     source_to_element_test.cpp \
     srtm_parser_test.cpp \
     tag_admixer_test.cpp \
     tesselator_test.cpp \
     triangles_tree_coding_test.cpp \
+
```

Comment:
No need in this blank line, no need in trailing '\' for the last entry in HEADERS.